core/servers/content/web_handlers/webfinger.js

```diff
@@ -77,17 +77,41 @@ exports.getModule = class WebFingerServerModule extends ServerModule {
         });
     }
 
+    _profileUrl(user) {
+        return this.webServer.instance.buildUrl(`/wf/@${user.username}`);
+    }
+
     _profilePageLink(user) {
-        const href = this.webServer.instance.buildUrl(`/wf/@${user.username}`);
+        const href = this._profileUrl(user);
         return {
             rel: 'http://webfinger.net/rel/profile-page',
             type: 'text/plain',
             href,
         };
     }
 
+    _selfUrl(user) {
+        return this.webServer.instance.buildUrl(`/users/${user.username}`);
```

Comment:
We'll need to decide where we want all these URLs to point to, and how much should be configurable. Since the web server can be used for arbitrary hosting, we'll want some of these to be less generic I think. 

I recently added `/_internal/` and `/_f/`, and `/wf/` above. I wonder if `_internal` should be renamed to e.g. `/_enig/`, and we stuff all the rest under that:
* `/_enig/pw/`: password resets
* `/_enig/2fa/`: 2FA/OTP stuff
* `/_enig/wf/`: WebFinger stuff
* `/_enig/ap/`: ActivityPub related stuff

Reply:
Maybe `/enigma-api/*` - Assuming ActivityPub isn't a standardized endpoint.